Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,17 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+const CardContent = () => {
+  return (
+    <div className="card-content">
+      <h2>Get started with React</h2>
+      <p class="cardContentP">
+        React makes it painless to create interactive UIs. Design simple views
+        for each state in your application.
+      </p>
+      <p class="reactLink">reactjs.org</p>
```

Comment:
```suggestion
      <a href="https://reactjs.org/" class="reactLink">reactjs.org</a>
```
But really the a should be around the entire card container